apps/core/lib/core/schema/repository.ex

```diff
@@ -198,6 +198,7 @@ defmodule Core.Schema.Repository do
     |> cast_assoc(:shell)
     |> foreign_key_constraint(:publisher_id)
     |> cast_assoc(:integration_resource_definition)
+    |> validate_format(:name, ~r/^[a-z][-a-z0-9]*$/, message: "name must be formatted as a valid kubernetes namespace")
```

Comment:
Namespace regex follows the RFC 1123 standard to match the name. Names like `1-ab3` are also valid. This is the official regex used by the kubectl to validate the namespace name: `[a-z0-9]([-a-z0-9]*[a-z0-9])?`.

Reply:
Ah cool, will change that